bot/exts/recruitment/talentpool/_review.py

```diff
@@ -66,26 +66,40 @@ def schedule_review(self, user_id: int) -> None:
             self._review_scheduler.schedule_at(review_at, user_id, self.post_review(user_id, update_database=True))
 
     async def post_review(self, user_id: int, update_database: bool) -> None:
-        """Format a generic review of a user and post it to the nomination voting channel."""
+        """Format the review of a user and post it to the nomination voting channel."""
+        review, seen_emoji = await self.make_review(user_id)
+        if not review:
+            return
+
+        guild = self.bot.get_guild(Guild.id)
+        channel = guild.get_channel(Channels.nomination_voting)
+
         log.trace(f"Posting the review of {user_id}")
+        message = (await self._bulk_send(channel, review))[-1]
+        if seen_emoji:
+            for reaction in (seen_emoji, "\U0001f44d", "\U0001f44e"):
```

Comment:
I think using `"\N{THUMBS UP SIGN}"` and `"\N{THUMBS DOWN SIGN}"` is a bit clearer for the emojis